src/promptflow/promptflow/batch/_base_executor_proxy.py

```diff
@@ -77,6 +78,7 @@ async def exec_line_async(
         inputs: Mapping[str, Any],
         index: Optional[int] = None,
         run_id: Optional[str] = None,
+        enable_stream_output=False,
```

Comment:
Pass enable_stream_output by create (**kwargs).

Reply:
updated. personally I hate hiding parameters in kwargs as developer will easily miss them; usually we use this to keep backward compatibility and hide parameters from public SDK. you may consider this in further PR